### PR TITLE
Add GUI double-click analogy 

### PR DESCRIPTION
**Pull request by Ephantus2017**
When introducing the 'cd' command I think it might be useful to compare it to double-clicking a folder in a graphical user interface as you would in a platform with such windows.

Please delete this line and the text below before submitting your contribution.

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.  

---

### CODE REVIEW

File: _episodes/02-filedir.md [hunk 1]
```diff
@@ -520,7 +520,7 @@ directory name to change our working directory.
 `cd` stands for 'change directory',
 which is a bit misleading:
 the command doesn't change the directory,
-it changes the shell's idea of what directory we are in.
+it changes the shell's idea of what directory we are in. The 'cd' command is akin to double clicking a folder in a graphical interface to get into a folder.
```
Comment (gcapes) [inline _episodes/02-filedir.md:523]: Please split this line after the full stop.

Comment (gcapes) [inline _episodes/02-filedir.md:523]: Please use backticks to mark up commands:
'cd' -> `cd`

Comment (Ephantus2017) [reply _episodes/02-filedir.md]: Done.

Comment (Ephantus2017) [reply _episodes/02-filedir.md]: Done.

**Pull request merged**